src/main/views/macro/date.njk

```diff
@@ -11,6 +11,11 @@
     {% if monthError %}<span class="error-message">{{ t(monthError) }}</span>{% endif %}
     {% if yearError %}<span class="error-message">{{ t(yearError) }}</span>{% endif %}
     {% if dateError %}<span class="error-message">{{ t(dateError) }}</span>{% endif %}
+    <fieldset id="dateofbirth[label]">
```

Comment:
This will cause ally test case failure, So please can you review this changes before merging this PR

Reply:
I have removed the fieldset to make the ally test case to pass. Please review my changes before proceeding further.

Reply:
I have removed the fieldset to make the ally test case to pass. Please review my changes before proceeding further.